.github/ISSUE_TEMPLATE/70_org_editor-checklist.md

```diff
@@ -0,0 +1,37 @@
+---
+name: "[Organizer -> Editor] Initial Editor Checklist"
+about: Checklist for Associate Editor to fill out prior to review
+title: "[PRE-REVIEW] Initial Editor Checklist"
+labels: pre-review
+assignees: ''
+---
+
+<!--
+## DO NOT EDIT THIS FILE OUTSIDE OF THE journalovi/jovi-workflows REPOSITORY
+##
+## This file is automatically updated in all repositories within the journalovi
+## Github organization whenever the version in journalovi/jovi-workflows is
+## changed, so any other edits will be overwritten. To update this file, make
+## a commit or pull request at https://github.com/journalovi/jovi-workflows
+-->
+
+### Instructions for Editor
+- If there is anything in the list below that you are unsure of, you can reply to this issue (after you created it) and tag the responsible organizers. Organizers' Github IDs are listed on [People page](https://www.journalovi.org/people.html).
+- Close this issue when all items are complete.
+
+### Check that the live version of the article contains at least the following information:
+- [ ] Author information
+- [ ] Abstract
+- [ ] Structure that gives readers an overview and facilitates navigation, such as:
+   - [ ] Table of contents (if the article is text-intensive)
+   - [ ] Link or buttons to different parts of the paper or switching between paper-mode vs. interactive mode
+   - [ ] Some articles (e.g,. narrative visualization) might lend itself to be progressively disclosed without an overview
+- [ ] The following required sections at the end of the paper, after References:
+   - [ ] Research materials
```

Comment:
The `Research materials` and `License` items are already in the **Open Materials Review** checklist. There is no need for AE to check them, so I suggest removing them from this checklist.